projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad.php

```diff
@@ -191,6 +191,14 @@ function wpcom_launchpad_get_task_list_definitions() {
 			),
 			'is_enabled_callback' => 'wpcom_launchpad_is_paid_newsletter_enabled',
 		),
+		'earn'                   => array(
+			'title'               => 'Earn',
+			'task_ids'            => array(
+				'stripe_connected',
+				'paid_offer_created',
+			),
+			'is_enabled_callback' => '__return_true',
```

Comment:
sure this doesn't make it show up somewhere else?

Reply:
No, I don't think there's anyplace where a launchpad checklist is automatically displayed based on this property.